Dockerfile

```diff
@@ -126,51 +126,51 @@ FROM base AS tomlv
 ENV INSTALL_BINARY_NAME=tomlv
 COPY hack/dockerfile/install/install.sh ./install.sh
 COPY hack/dockerfile/install/$INSTALL_BINARY_NAME.installer ./
-RUN PREFIX=/opt/$INSTALL_BINARY_NAME ./install.sh $INSTALL_BINARY_NAME
+RUN PREFIX=/build/$INSTALL_BINARY_NAME ./install.sh $INSTALL_BINARY_NAME
```

Comment:
This should be `/build/` (without `$INSTALL_BINARY_NAME`) because it's the target-directory